code/parse/sexp.h

```diff
@@ -1264,6 +1384,15 @@ bool generate_special_explosion_block_variables();
 int num_block_variables();
 bool has_special_explosion_block_index(ship *shipp, int *index);
 
+// sexp_containers
+// DISCUSSME: rename to sexp_get_container_index()?
+int get_sexp_container_index(const char* name);
```

Comment:
I guess the name of this function should be `sexp_get_sexp_container_index()`? I think it was originally something like that, but I didn't understand why, so I got rid of it.

Ditto that it should presumably be `sexp_update_sexo_containers()` below.

Reply:
I've always tended to use the convention that functions that start with sexp are actual SEXPs. That one seems to be mostly respected in the code, but there are a few exceptions.